HUD/GetScaleformMovieCursorSelection.md

```diff
@@ -6,10 +6,10 @@ aliases: ["0x632B2940C67F4EA9"]
  
 ```c
 // 0x632B2940C67F4EA9
-BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* slotIndex, int* p4);
```

Comment:
Removing parameters is not supported for backwards compatibility reasons.

Reply:
Unfortunately that's how it'll have to be, unless blattersturm thinks otherwise.

Reply:
> 

i see 🤔